app/src/main/res/layout/fragment_review_and_confirm.xml

```diff
@@ -318,10 +318,11 @@
         android:text="@string/voter_agreement_text"
         />
 
-    <Button
```

Comment:
There were some issues around the coloring and elevation when it came to matching the previous implementation. This mostly fixes that. The only difference being that the `disabled` state of the button uses `colorAccent` as opposed to the white background in the original implementation.

This is just to avoid having to create a separate drawable to handle two different background tints as the original implementation did. If the color was a requirement, it's too easy to add that back in